pyccel/commands/pyccel_wrap.py

```diff
@@ -89,105 +58,87 @@ def pyccel_wrap_command() -> None:
-        args.conda_warnings = 'basic'
+    parser = argparse.ArgumentParser(description="Pyccel's command line interface.",
+                      add_help=True)
 
+    #... Help and Version
+    add_version_flag(parser)
+    # ...
     # ...
-    # ... report error
-    if filename.is_file():
-        fext = filename.suffix
-        if fext != '.pyi':
-            errors = Errors()
-            # severity is error to avoid needing to catch exception
-            errors.report(INVALID_FILE_EXTENSION,
-                            symbol=fext,
-                            severity='error')
-            errors.check()
-            sys.exit(1)
-    else:
-        # we use Pyccel error manager, although we can do it in other ways
-        errors = Errors()
-        # severity is error to avoid needing to catch exception
-        errors.report(INVALID_FILE_DIRECTORY,
-                        symbol=filename,
-                        severity='error')
-        errors.check()
-        sys.exit(1)
+    setup_pyccel_wrap_parser(parser)
+    # ...
+    args = parser.parse_args()
     # ...
 
-    accelerators = []
-    if mpi:
-        accelerators.append("mpi")
-    if openmp:
-        accelerators.append("openmp")
-    if openacc:
-        accelerators.append("openacc")
+    print("warning: The pyccel-wrap command is deprecated and will be removed in v2.3. Please use pyccel wrap.", file=sys.stderr)
 
-    # ...
+    from pyccel.errors.errors     import Errors, PyccelError
+    from pyccel.utilities.stage   import PyccelStage
 
-    # ...
-    # this will initialize the singleton ErrorsMode
-    # making this setting available everywhere
-    err_mode = ErrorsMode()
-    if args.developer_mode:
-        err_mode.set_mode('developer')
-    else:
-        err_mode.set_mode(os.environ.get('PYCCEL_ERROR_MODE', 'user'))
-    # ...
+    pyccel_stage = PyccelStage()
+    errors = Errors()
 
     try:
-        # TODO: prune options
-        execute_pyccel_wrap(filename,
-                       convert_only    = args.convert_only,
-                       verbose         = args.verbose,
-                       time_execution  = args.time_execution,
-                       language        = args.language.lower(),
-                       compiler_family = str(compiler) if compiler is not None else None,
-                       debug           = args.debug,
-                       accelerators    = accelerators,
-                       folder          = output if output is not None else None,
-                       output_name     = filename.stem,
-                       conda_warnings  = args.conda_warnings)
+        pyccel_wrap(**vars(args))
     except PyccelError:
-        sys.exit(1)
+        pass
+
+    pyccel_stage.pyccel_finished()
+    print(errors, end='')
+    sys.exit(errors.has_errors())
+
+def pyccel_wrap(*, filename, language, output, **kwargs) -> None:
+    """
+    Call the pyccel wrap pipeline.
+
+    Import and call the pyccel wrap pipeline.
```

Comment:
For clarity I suggest quoting `pyccel wrap`:

```suggestion
    Call the `pyccel wrap` pipeline.

    Import and call the `pyccel wrap` pipeline.
```